### PR TITLE
Help page

### PR DESCRIPTION
**Pull request by jdtoombs**
- Allow the help desk to tell the difference between the map, and parcel detail view by checking if the sidebar is open
- Help desk added for Disposition of projects, and assessment (will need some wording checks here!)

### CODE REVIEW

File: frontend/src/features/help/components/text/CreateDisposalNavigationHelpText.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+/**
+ * Help Text for the creation of disposal project.
+ */
+export const CreateDisposalNavigationHelpText = () => {
+  return (
+    <p>
+      A stepper is provided to guide you through the disposal process. You will not be able to
+      advance to the next step until validation has passed for the current step.
+    </p>
```
Comment (agalbrai) [inline frontend/src/features/help/components/text/CreateDisposalNavigationHelpText.tsx:11]: I know this is just a first pass so not a huge deal, but I don't think non-tech-y people will quite understand what "validation" entails

File: frontend/src/features/help/containers/HelpModalContentContainer.tsx [hunk 2]
```diff
@@ -18,6 +23,10 @@ interface IHelpModalContentContainerProps {
 const getHelpPageByLocation = (location: any): IHelpPage | undefined => {
   let helpPage: IHelpPage | undefined = undefined;
   helpPages.forEach((page, path) => {
+    /** need to check paramaters here outside of the pathname as mapview and parcel detail form now share same pathname */
+    if (location.search.includes(HelpPageKeys.PROPERTY_DETAIL_PAGE)) {
+      helpPage = PropertyDetailViewHelpPage;
```
Comment (jdtoombs) [inline frontend/src/features/help/containers/HelpModalContentContainer.tsx:28]: Have to check based off the params in order to see the sidebar. Which means we cannot use `location.pathname` but `location.search` for this scenario

**Review by agalbrai — APPROVED** [review]

File: frontend/src/features/help/components/text/CreateDisposalNavigationHelpText.tsx (hunk 1 above)
Comment (jdtoombs) [reply frontend/src/features/help/components/text/CreateDisposalNavigationHelpText.tsx:11]: Kinda had that thought in my head too..

File: frontend/src/features/help/components/HelpModal.tsx [hunk 3]
```diff
@@ -37,6 +37,11 @@ const ModalHeader = styled(Modal.Header)`
   padding: 5px;
 `;
 
+const StyledTooltip = styled(TooltipIcon)`
+  margin-left: 5px;
+  margin-bottom: 5px;
+`;
+
```
Comment (jdtoombs) [inline frontend/src/features/help/components/HelpModal.tsx:44]: Tooltip icon was slightly misaligned

File: frontend/src/features/help/components/HelpSubmitBox.scss [hunk 4]
```diff
@@ -3,7 +3,7 @@
     display: flex;
     label {
       padding-right: 1rem;
-      width: 20%;
+      width: 40%;
```
Comment (jdtoombs) [inline frontend/src/features/help/components/HelpSubmitBox.scss:6]: Text boxes were slightly misaligned due to larger label sizes

**Review by Fosol — APPROVED** [review]

**Pull request merged**